### ISSUE TITLE
make `Signal.remove()` accept a `SignalBinding` as well

### ISSUE THREAD

**Issue by millermedeiros** [millermedeiros/js-signals#7]

**Comment by millermedeiros** [comment]
not a good idea, will have to check if it is an Object or Function all the times (unnecessarily) and if you have a reference to the `SignalBinding` you can just call `SignalBinding.detach()` (way easier)

PS: this entry was just to remind myself why I created the `detach` method, I'm sure in a couple months someone would ask for the same thing and I wouldn't remind why I didn't implemented it.
